ontology/nomen.md

```diff
@@ -3,7 +3,7 @@ layout: ontology_detail
 id: nomen
 type: owl:Ontology
 label: NOMEN
-title: NOMEN - A nomenclatural ontology for biological names
+title: A nomenclatural ontology for biological names
```

Comment:
Wouldnt make that change.. The title is a bit at the discretion of the editors.

Reply:
Alright will open a different issue for this. I am very against the redundant usage of acronyms in the title since it's wasting space and increasing possibility for confusion. For example, in the case of UBERON, there is a self-referential use of UBERON in the title.

Reply:
Reverted in 4dad270